core/src/test/java/org/infinispan/distexec/DistributedExecutorFailoverTest.java

```diff
@@ -124,4 +160,25 @@ public int maxFailoverAttempts() {
          return maxFailoverCount;
       }
    }
+
+   static class SameNodeTaskFailoverPolicy implements DistributedTaskFailoverPolicy {
+      private final int maxFailoverCount;
```

Comment:
Counters are going to come in Infinispan 9.1. https://github.com/infinispan/infinispan/pull/4350